packages/react-native-web/src/exports/ScrollView/ScrollViewBase.js

```diff
@@ -178,11 +188,17 @@ export default class ScrollViewBase extends Component<*> {
   }
 
   _handleScrollEnd(e: Object) {
-    const { onScroll } = this.props;
+    const { onScroll, onScrollEndDrag, onMomentumScrollEnd } = this.props;
     this._state.isScrolling = false;
     if (onScroll) {
       onScroll(normalizeScrollEvent(e));
     }
+    if (onScrollEndDrag) {
+      onScrollEndDrag(normalizeScrollEvent(e));
```

Comment:
Are these events really the same in React Native? I suspect 'drag' is meant to end when you lift your finger, but momentum continues until the scroll stops.

Reply:
hi @necolas , `onscrollbegindrag` be listed in `rn 0.57`'s doc,

https://facebook.github.io/react-native/docs/scrollview#onscrollbegindrag

could you push this PR forward ?

@gwuhaolin looks like give up this PR, but 

> I'm not merging something incomplete and with no response from the author

so, could you give some update?